src/modules/resource/monitor.c

```diff
@@ -169,7 +176,7 @@ static void broker_online_cb (flux_future_t *f, void *arg)
     if (!(up = group_get (f))) {
         flux_log (h,
                   LOG_ERR,
-                  "monitor: group.get: %s",
+                  "monitor: broker.online: %s",
```

Comment:
This seems like it was meant for a different commit? (Is now in `resource: monitor broker.torpid `)